src/components/Common.js

```diff
@@ -86,4 +86,46 @@ export const DetailColumn = styled.ul`
 
 export const DetailAnswer = styled(P)`
   margin-bottom: 0.85em;
-`
\ No newline at end of file
+`
+// given array, split into array of chunks of size n 
+// "balanced" (subarrays' lengths differ as less as possible) or 
+// "even" (all subarrays but the last have the same length):
+// https://stackoverflow.com/questions/8188548/splitting-a-js-array-into-n-arrays
+export const chunkify = ((a, n, balanced) => {
```

Comment:
I'd out this in `src/utility` as it's not really a component

Reply:
mmm something like utilities.js? not sure what to call it

Reply:
would've preferred `index.js` but im too slow 🐢